src/components/buttons/button.tsx

```diff
@@ -1,29 +1,45 @@
 import React from 'react';
 import styled from 'styled-components';
-import { TextStyles } from '../typography/typography-v2';
+import { Icon, IconSize } from '../icon/icon';
+import { resetButton } from '../utils';
+import { lighten } from 'polished';
 
 const StyledButton = styled.button`
-  height: 40px;
+  ${resetButton};
+  color: #ffffff;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 40px;
+  border-radius: 30px;
+  vertical-align: middle;
+  cursor: pointer;
 
   background: linear-gradient(275.41deg, #543fd7 0%, #2756fd 100%);
-  border-radius: 30px;
-  border: none;
```

Comment:
Why did you decide to move these two properties up between the other props but leave the `background-color` down here, separated by a new line? Is there a reason for this or just personal preference? The newline confuses me (I know that it existed before - it was equally as confusing back then :joy:) and the change itself makes the review more complicated than it would need to be.